client/src/components/Search/Search.js

```diff
@@ -9,27 +9,36 @@ const Search = props => {
   const [error, setError] = useState(null);
   const [loading, setLoading] = useState(true);
   const [searchResult, setSearchResult] = useState(null);
-  const { term } = props.match.params;
+  const [term, setTerm] = useState(props.match.params.term);
+  const reqTime = React.useRef(0);
   const apiBase = 'https://www.udemy.com/api-2.0';
+  console.log(term);
```

Comment:
no console logs for frontend.

also please add new lines between lines of code.